language/move-vm/types/src/values/values_impl.rs

```diff
@@ -1884,6 +1884,188 @@ impl Display for Locals {
     }
 }
 
+pub mod debug {
+    use super::*;
```

Comment:
This module feels like it could/should live in its own file unless the visibility rules of `values_impl` prevent it.

Reply:
The current visibility rules require this to be a sub-module of values_impl, but yeah it can live in its own file. I'll take a look at a later time and see what will be the best module & directory structure for move_vm/types.